netxQueries/fullRecordSingleSync.sql

```diff
@@ -30,6 +30,8 @@ CROSS JOIN (
 		constituent_records
 	INNER JOIN object_constituent_mappings ON
 		constituent_records."constituentID" = object_constituent_mappings."constituentRecordId"
+		AND
+		constituent_records."objectId" = object_constituent_mappings."objectId"
```

Comment:
Each constituent record now has an object_id because constituent records are actually unique to each object so we need this change when pulling a full record